src/components/Routes/styles.module.sass

```diff
@@ -0,0 +1,26 @@
+.nav
+  align-self: center
+  position: absolute
+  right: 20px
+  @media screen and (max-width: 530px)
+    position: initial
+
+.list
+  display: flex
+  flex-direction: row
+  @media screen and (max-width: 530px)
+    padding: 10px
+    > li
```

Comment:
IT's not a good practice to not add a class name for elements suing custom styles. You should have used either styles modules or BEM for both `li` and `a` elements